Lib/concurrent/futures/_base.py

```diff
@@ -50,10 +50,7 @@ class CancelledError(Error):
     """The Future was cancelled."""
     pass
 
-class TimeoutError(Error):
-    """The operation exceeded the given deadline."""
-    pass
-
+TimeoutError = TimeoutError  # make local alias for the standard exception
```

Comment:
Same remark, can it be moved to ``__init__.py`` and mention the bpo in the comment?